client/App.js

```diff
@@ -22,36 +23,14 @@ import reducers from './app/redux/reducers';
 // Specific Redux actions we need to call on app startup
 import { saveUserNotificationID } from './app/redux/actions/UserInformationActions';
 import { getFavorites } from './app/redux/actions/FavoritesActions';
+import BottomTabs from './app/components/BottomTabs';
 
 // Configuring logger for the state of our app
 const loggerMiddleware = createLogger({ predicate: (getState, action) => __DEV__ });
 
 // Configuring our navigation middleware
-const transitionConfig = () => {
```

Comment:
This just removes the problematic transitions? Can we look into adding them back at some point now that we have the stack navigator?